lib/WP_Auth0_Options.php

```diff
@@ -207,9 +207,9 @@ protected function defaults() {
 
 			// Features
 			'password_policy'           => 'fair',
-			'sso'                       => false,
-			'singlelogout'              => false,
-			'override_wp_avatars'       => true,
+			'sso'                       => 0,
```

Comment:
The settings are validated and checked for `int` 1 or 0. This adjusts the default value (used when the plugin is first installed) to use the same.

Reply:
Maybe add a comment to say 0=false, 1=true or is it well known in the PHP world?